testing/samurai/APKBUILD

```diff
@@ -0,0 +1,27 @@
+# Contributor: TBK <alpine@jjtc.eu>
+# Maintainer: TBK <alpine@jjtc.eu>
+pkgname=samurai
+pkgver=0.3
+pkgrel=0
+pkgdesc="Ninja-compatible build tool written in C99 with focus on simplicity, speed and portability."
+url="https://github.com/michaelforney/samurai"
+arch="all"
+license="GPL-2.0"
```

Comment:
The [LICENSE](https://github.com/michaelforney/samurai/blob/master/LICENSE) file lists a few BSD/MIT-like licenses (I haven't checked exactly) and an Apache licence 2.0, but no GPL.